connect/mirror/src/main/java/org/apache/kafka/connect/mirror/DefaultConfigPropertyFilter.java

```diff
@@ -60,6 +65,11 @@ public boolean shouldReplicateConfigProperty(String prop) {
         return !excluded(prop);
     }
 
+    @Override
+    public boolean shouldReplicateSourceDefault(String prop) {
+        return useDefaultsFrom.equals("source");
```

Comment:
We should check against the `config.properties.exclude` pattern as well.

Reply:
(Depends on discussion on a different thread but for now I still believe this part needs to be changed)